docs/architecture/_index.md

```diff
@@ -0,0 +1,192 @@
+This peer-to-peer connectivity is enabled by sending [gRPC commands over WebRTC connections](/architecture/machine-to-machine-comms/#low-level-inter-robotsdk-communication).
+
+On startup, `viam-server` establishes a {{< glossary_tooltip term_id="webrtc" text="WebRTC" >}} connection with the [Viam app](https://app.viam.com).
+`viam-server` pulls its configuration from the app, caches it locally, and initializes all components and services based on that configuration.
+
+If [sub-parts or remote parts](#complex-machines-with-multiple-parts) are configured, communications are established between the `viam-server` instances on each of them.
+
+If you have client code running on a separate computer, that code sends API requests to `viam-server` using gRPC over WebRTC.
+If a WebRTC connection cannot be established, the request is sent directly over gRPC.
+When a built-in service communicates with a component, for example when the vision service requests an image from a camera, `viam-server` handles that request as well.
+
+When you control your machine or view its camera streams or sensor outputs from the Viam app **CONTROL** tab, those connections happen over WebRTC.
+The Viam app hits the same API endpoints as your SDK client code, with `viam-server` handling requests.
+
+{{% alert title="Protobuf APIs" color="info" %}}
+All Viam APIs are defined with the [Protocol Buffers (protobuf)](https://protobuf.dev/) framework.
+This is what enables Viam to provide SDKs in a variety of different programming languages.
+{{% /alert %}}
+
+For more details, see [Machine-to-Machine Communication](/architecture/machine-to-machine-comms/).
+
+### Security
+
+TLS certificates certificates provided by the Viam app ensure that all communication is authenticated and encrypted.
+
+Viam uses API keys with [role-based access control (RBAC)](/fleet/rbac/) to control access to machines from client code.
+
+## Data management flow
+
+{{<imgproc src="/architecture/data-flow.svg" resize="x1100" declaredimensions=true alt="Data flowing from local disk to cloud to the Viam app, SDKs, and MQL and SQL queries." >}}
+<br>
+
+Here's how data flows in Viam:
+
+1. Data collected by your components, such as sensors and cameras, is first stored locally in a specified directory (defaults to <file>~/.viam/capture</file>).
+   You control how often to capture data and where to store it using the configuration file.
+
+   - You can also sync data from other sources by putting it into this folder.
+     <br><br>
+
+1. `viam-server` syncs data to a MongoDB database in the cloud at your specified interval, and deletes the data from the local directory.
+
+1. You can view your data from the Viam app or query it using Viam SDKs, MQL, or SQL.
+
+If a device has intermittent internet connectivity, no data is lost because it is stored locally until the machine can reconnect to the cloud.
```

Comment:
That's not accurate. Eventually the disk is full and data is lost, so how about
```suggestion
If a device has intermittent internet connectivity, datais stored locally until the machine can reconnect to the cloud.
```

That is accurate without making the same guarantee

Reply:
```suggestion
If a device has intermittent internet connectivity, data is stored locally until the machine can reconnect to the cloud.
```